api/src/main/java/com/example/api/error/exception/MissingProfessorFeedbackAttributeException.java

```diff
@@ -0,0 +1,14 @@
+package com.example.api.error.exception;
+
+public class MissingProfessorFeedbackAttributeException extends Exception{
```

Comment:
Powinnismy bardziej uoglnic te exceptiony, w sensie dac ogolny exception ze MissingAttributeExceptiion i dawac w msg o co chodzi bo takto nam sie tego juz nasralo, w sensie nie mowie ze to tylok Ty, ale ja tez tak zrobilem i ostatnio mialem rozkmine zeby to jakos ogarnac, ale narazie mysle ze moze zostac

Reply:
zrobione